src/FirstSteps/RPG/AdventuresRegistry.cs

```diff
@@ -0,0 +1,40 @@
+﻿using FirstSteps.RPG.Adventures;
+using Spectre.Console;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace FirstSteps.RPG
+{
+    public class AdventuresRegistry 
+    {
+        
+        private static List<IAdventure> _adventures = new List<IAdventure>()
+        {
+          new Dungeons(),
+          new Forest(),
+          new Hell(),
+          new Mine(),
+          new TreasureChest(),
+          new Village()
+        };
+        
+        public static string GetCommands()
```

Comment:
Follow the issue description. `GetCommnads` should return a **list of all available adventure commands**. You can take them from the `_adventures` list.

Reply:
The [Select](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.select?view=net-7.0#system-linq-enumerable-select-2(system-collections-generic-ienumerable((-0))-system-func((-0-system-int32-1)))) method can be useful.